prisma/schema.prisma

```diff
@@ -222,3 +222,11 @@ model Deposit {
   @@map("deposits")
   @@unique([transaction_hash, graffiti], name: "uq_deposits_on_transaction_hash_and_graffiti", map: "uq_deposits_on_transaction_hash_and_graffiti")
 }
+
+model DepositHead {
+    id                Int                @id
+    block_hash        String             @db.VarChar
+    block_sequence    Int
+
+    @@map("deposithead")
```

Comment:
```suggestion
    @@map("deposit_head")
```

Reply:
I think this comment was on the old version, but it was already done on the non WIP.